go.mod

```diff
@@ -266,3 +266,5 @@ replace (
 	github.com/pingcap/tidb/parser => ./parser
 	go.opencensus.io => go.opencensus.io v0.23.1-0.20220331163232-052120675fac
 )
+
+replace github.com/tikv/client-go/v2 => github.com/crazycs520/client-go/v2 v2.0.0-alpha.0.20230609110852-067ec9341f7b
```

Comment:
Please run ```make bazel_prepare```.

Reply:
I will update this after https://github.com/tikv/client-go/pull/833 merged.